odd-platform-ui/src/components/Alerts/AlertsList/AlertsList.tsx

```diff
@@ -47,6 +47,13 @@ const AlertsList: React.FC<AlertsListProps> = ({
     fetchNextPage();
   }, [fetchAlerts]);
```

Comment:
Could you also remove old useEffect? It turns out that we duplicated requests.

Reply:
done